### DISCUSSION TITLE
Does this Bulb support Binding/Groups?

### DISCUSSION THREAD

**Discussion by ooohaaa** [Koenkk/zigbee2mqtt#9871] [Devices]
Hi all,
this is my curent setup:
- Two of these bulb: [HORNBACH 10297667](https://www.zigbee2mqtt.io/devices/10297667.html#hornbach-10297667)
- One of this reomote: [IKEA E1524/E1810](https://www.zigbee2mqtt.io/devices/E1524_E1810.html#ikea-e1524%252Fe1810)

I was able to bind the remote (Cluster 'LevelCtrl' and 'OnOff') to the bulbs directly. This works fine.

I would like to bind the remote to a group (which contains these two bulbs). I can add the binding, but the bulbs don't react to the commands of the remote.

Do these bulbs support "Group-Bindig"?
What could be wrong?

**Comment by sjorge** [comment]
It's actually the remote that no longer supports group binding, IKEA remove support for this on the latest firmware, IIRC 2.3.014 is the last version that support binding to groups. See https://github.com/Koenkk/zigbee2mqtt/issues/7716

**Reply by ooohaaa** [reply]
Oh, that's bad news. Thanks for your reply anyway!
